.hlint.yaml

```diff
@@ -337,6 +597,15 @@
 - warn: { name: "Use 'atomicWriteIORef' from Universum"
         , lhs: Data.IORef.atomicWriteIORef, rhs: Universum.atomicWriteIORef }
 
+## String
+
+- warn: { name: "Use 'ByteString' from Universum"
+        , lhs: Data.ByteString.ByteString, rhs: Universum.ByteString }
+- warn: { name: "Use 'IsString' from Universum"
+        , lhs: Data.String.IsString, rhs: Universum.IsString }
+
+- warn: { name: "Use 'Text' from Universum"
+        , lhs: Data.Text.Text, rhs: Universum.Text }
```

Comment:
I also want to add rules for `LText` and `LByteString` instead of `Data.Text.Lazy.Text` and similar.